tools/run_local_ironic.sh

```diff
@@ -93,7 +93,7 @@ data:
 kind: Secret
 metadata:
    name: ironic-cacert
-   namespace: capm3-system
+   namespace: ${IRONIC_NAMESPACE}
```

Comment:
What a different approach here?

Reply:
If I am not mistaking ${IRONIC_NAMESPACE} was used during pivoting to define in which namespace we want to move Ironic.

Reply:
This PR is dependent on two other patches, one in m3-dev-env and other in CAPM3. 

1. First one, [here](https://github.com/metal3-io/metal3-dev-env/pull/721/files#diff-cfcb1fb44601d83d47282622bdbd46c08d3278b4a6f6a0da4b2903ba609885dfR176-R180) sets the namespace for the ironic depending on the API version of CAPM3, where in CAPM3 v1a4 we deploy BMO as part of it and namespace is `capm3-system`, whereas in CAPM3 v1a5 we are trying to decouple BMO from CAPM3, the namespace is `baremetal-operator-system`. So, depending on the which ever CAPM3 API version is used we set namespace accordingly. 
2. Second one, is an effort to decouple BMO from CAPM3 and you can check the PR [here](https://github.com/metal3-io/cluster-api-provider-metal3/pull/242), which takes BMO out from CAPM3 in CAPM3 v1a5.

m3-dev-env [PR721](https://github.com/metal3-io/metal3-dev-env/pull/721) is testing both changes from this PR and CAPM3 one in CI.

Reply:
@dtantsur using `${IRONIC_NAMESPACE}` and `${NAMEPREFIX}-system` are similar. However, I agree that we should make it synchronize with the others. Will fix it. Thanks

Reply:
Okay, understood. I assume this patch is not ready to merge yet, because the metal3-dev-env change that defined IRONIC_NAMESPACE hasn't merged?

Reply:
@dtantsur this can already go in, IRONIC_NAMESPACE is defined already and set to `capm3-system` as default in metal3-dev-env master, it will be set in 721 accordingly based on CAPM3 API version.